go/cmd/vtcombo/main.go

```diff
@@ -61,7 +61,7 @@ var (
 	mysqlPort          = flag.Int("mysql_port", 3306, "mysql port")
 	externalTopoServer = flag.Bool("external_topo_server", false, "Should vtcombo use an external topology server instead of starting its own in-memory topology server. "+
 		"If true, vtcombo will use the flags defined in topo/server.go to open topo server")
-	plannerVersion           = flag.String("planner-version", "gen4", "Sets the default planner to use when the session has not changed it. Valid values are: V3, Gen4, Gen4Greedy and Gen4Fallback. Gen4Fallback tries the gen4 planner and falls back to the V3 planner if the gen4 fails.")
```

Comment:
This makes a lot of sense. Otherwise users would have provided the old flag, and it would have conflicted with the default in this flag!